pycrdt_websocket/yroom.py

```diff
@@ -138,30 +144,42 @@ async def _broadcast_updates(self):
                 # broadcast internal ydoc's update to all clients, that includes changes from the
                 # clients and changes from the backend (out-of-band changes)
                 for client in self.clients:
-                    self.log.debug("Sending Y update to client with endpoint: %s", client.path)
-                    message = create_update_message(update)
-                    self._task_group.start_soon(client.send, message)
+                    try:
+                        self.log.debug("Sending Y update to client with endpoint: %s", client.path)
+                        message = create_update_message(update)
+                        self._task_group.start_soon(client.send, message)
+                    except Exception as exception:
+                        self._handle_exception(exception)
                 if self.ystore:
-                    self.log.debug("Writing Y update to YStore")
-                    self._task_group.start_soon(self.ystore.write, update)
+                    try:
+                        self._task_group.start_soon(self.ystore.write, update)
+                        self.log.debug("Writing Y update to YStore")
+                    except Exception as exception:
+                        self._handle_exception(exception)
```

Comment:
I think that a follow-up to this PR would be to not handle exceptions here, but have a YStore have an optional exception handler, and do the handling there.

Reply:
Good point, @davidbrochart. I've created the following issue to track: https://github.com/jupyter-server/pycrdt-websocket/issues/36